benchmark/quantum-hilbert-matrix.py

```diff
@@ -0,0 +1,40 @@
+import cvxpy as cp
+import numpy as np
+
+
+class QuantumHilbertMatrix:
+    """
+    Composition of two quantum channels taken from equation #11 of https://arxiv.org/pdf/0904.4483.pdf
+    """
+
+    def setup(self):
+        # Import packages.
+        import cvxpy as cp
+        import numpy as np
+        from scipy import sparse
+
+        d = 2 # local hilbert space dimension
+        N = 3 # number of qubits
+        dim = d**N # hilbert space dimension of full system
+
+        I = sparse.identity(dim) # sparse identity matrix
+        # sparse matrices with the same density as the ones in my problem
+        rhs = sparse.random(m=dim**2,n=dim**2, density=0.05810546875)
+        A = sparse.random(m=dim**2,n=dim**2, density=0.29058837890625)
```

Comment:
We should fix the seed here.

Reply:
Done in [b84c927](https://github.com/cvxpy/benchmarks/pull/20/commits/b84c927f06e550f5f7877cca8b232d8eac000c3e)